agent/agents/postgres/pgstatmonitor/stat_monitor_cache.go

```diff
@@ -91,10 +91,11 @@ func (ssc *statMonitorCache) getStatMonitorExtended(ctx context.Context, q *refo
 
 	row, view := NewPgStatMonitorStructs(pgMonitorVersion)
 	conditions := "WHERE queryid IS NOT NULL AND query IS NOT NULL"
-	if pgMonitorVersion >= pgStatMonitorVersion09 {
-		// only pg_stat_monitor 0.9.0 and above supports state_code. It tells what is the query's current state.
+	if pgMonitorVersion >= pgStatMonitorVersion09 && pgMonitorVersion <= pgStatMonitorVersion11PG14 && prerelease != "" {
```

Comment:
I thought that we decided to have a check like `version >= 0.9 and version <= 2.0`

Reply:
Changed.